### ISSUE TITLE
data: Swedish COVID-19 Booster doses appear to be too high / wrong.

### ISSUE THREAD

**Issue by USMortality** [owid/covid-19-data#2817]
### Country

Sweden

### Domain

Vaccinations

### Which data is inaccurate or missing?

The reported booster doses at OWID does not appear right for Sweden.

OWID covid explorer currently lists latest proportion of population as vaccinated:
- 1st: 74%
- 2nd: 72%
- 3rd (boosted): 93%
It's already obvious from these numbers, that the 3rd is incorrect, as the 3rd dose is typically much less than the 2nd in other countries.

The official Swedish website, lists the 18+ aged population boosted at 66.6%, which should translate to a lower total population percentage, maybe around 40-60% or so...
<img width="1013" alt="Screenshot 2023-10-04 at 4 11 23 PM" src="https://github.com/owid/covid-19-data/assets/94856880/ee4723cb-cc11-4968-9f4b-9fcc510660ac">
https://www.folkhalsomyndigheten.se/folkhalsorapportering-statistik/statistikdatabaser-och-visualisering/vaccinationsstatistik/statistik-for-vaccination-mot-covid-19/

Appreciate, if anyone could check this. Thanks

### Why do you think the data is inaccurate or missing?

It's possible, that the cumulative doses were used in here?
https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/country_data/Sweden.csv

**Comment by USMortality** [comment]
As to my calcualtions it should be ~
53%

<img width="901" alt="Screenshot 2023-10-04 at 5 59 19 PM" src="https://github.com/owid/covid-19-data/assets/94856880/f2f8494e-d1c9-4dce-b251-14e0668fba7b">

<img width="627" alt="Screenshot 2023-10-04 at 6 08 44 PM" src="https://github.com/owid/covid-19-data/assets/94856880/2ae4576a-d6bc-4d1e-af20-e03fbef917ac">

**Comment by marcopiani** [comment]
OWID does not report the % of boosted. It reports

> Total number of vaccine booster doses administered, divided by the total population of the country. Booster doses are doses administered beyond those prescribed by the original vaccination protocol.

Given that many people got more than one booster, the total number of vaccine booster doses administered per capita is higher than the percentage boosted. No claim is made that what shown is the latter, and no claim is made that the doses reported are just 3rd doses.
![image](https://github.com/owid/covid-19-data/assets/18381804/8bee52ca-2329-48cc-b951-1bceea73e26b)

**Comment by lucasrodes** [comment]
Hi @USMortality,
Thanks for your message.

@marcopiani already addressed this correctly (thanks!). Our figures on the number of booster doses measure **(any) booster doses** and not **people with third doses**.


Longer explanation:

For the first dose, [this chart](https://ourworldindata.org/explorers/coronavirus-data-explorer?facet=none&country=~SWE&Metric=People+vaccinated&Interval=Cumulative&Relative+to+Population=true&Color+by+test+positivity=false) shows the number of **people** vaccinated with at least one dose. Similarly, [this chart](https://ourworldindata.org/explorers/coronavirus-data-explorer?facet=none&country=~SWE&Metric=People+fully+vaccinated&Interval=Cumulative&Relative+to+Population=true&Color+by+test+positivity=false) indicates the number of people vaccinated with all required doses in the initial protocol (typically two doses).

However, [this chart](https://ourworldindata.org/explorers/coronavirus-data-explorer?facet=none&country=~SWE&Metric=Vaccine+booster+doses&Interval=Cumulative&Relative+to+Population=true&Color+by+test+positivity=false) is showing **the number of booster doses** (and not people with booster doses). Note that the same individual may have received a third, fourth, ... dose.


In summary: In your initial comment, you compare the "number of people vaccinated" with "number of doses administered". Unfortunately, we do not have the figures on "people vaccinated with _N_ doses" where _N_ is greater than two.
